pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -174,6 +176,7 @@ func (r *ReconcileChaosEngine) Reconcile(request reconcile.Request) (reconcile.R
 			return reconcile.Result{}, nil
 		}
 	}
+
```

Comment:
Any logic behind adding this extra line? (also line no: 194)

Reply:
done @chandankumar4

Reply:
Still this line is here!!